state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -425,16 +434,18 @@ def timeon(self):
         """ return the time on a monotonic clock """
         return int(time.monotonic()) - self.BOOTTIME
 
-    def reset_boot_count(self):
-        """ reset boot count in non-volatile memory (nvm) """
-        self.c_boot = 0
-
-    def incr_logfail_count(self):
-        """ increment logfail count in non-volatile memory (nvm) """
-        self.c_logfail += 1
+    def reset_nvm(self):
```

Comment:
Reset nvm, doesn't reset NVM.

Reply:
It resets certain counters and flags, there should be a way to zero all of NVM.

Reply:
Why would we want to zero the parts of NVM that we aren't using?